starter-code/src/viking.js

```diff
@@ -1,5 +1,16 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+  attack() {
+    return this.strength;
+  }
+  receiveDamage(theDamage) {
+    this.health = this.health - theDamage;
+  }
+}
 
 // Viking
 class Viking {}
```

Comment:
Tu as été bloquée pour la suite ? Je te donne un exemple pour la class Viking que tu pourras utiliser pour la class Saxon aussi : 
 class Viking extends Soldier {
  constructor(name,health,strength){
    super(health,strength);
    this.name = name;
  }

  receiveDamage(damage){
    this.health -= damage;
    if (this.health>0){
      return (this.name+" has received " + damage.toString() + " points of damage");
    }
    return (this.name + " has died in act of combat")
  }

  battleCry(){
    return("Odin Owns You All!");
  }

}